active/gridsyntax/GridSyntaxSpec.md

```diff
@@ -0,0 +1,324 @@
+          <ColumnDefinition Width="300" />
+    </Grid.ColumnDefinitions>
+    <Grid.RowDefinitions>
+          <RowDefinition Height="1*" />
+          <RowDefinition Height="Auto" />
+          <RowDefinition Height="25" />
+          <RowDefinition Height= "14" />
+          <RowDefinition Height="20" />
+    </Grid.RowDefinitions>
+</Grid>
+```
+
+This code creates multiple ColumnDefinition objects and RowDefinitions objects and adds them to the Grid's ColumnDefinitions and RowDefinitions collections, respectively. ColumnDefinitions and RowDefinitions are read-only properties of Grid, and the syntax shown above is currently the only way to update these properties. Due to their read-only nature, existing typeconverters cannot be used with these properties. 
+
+The existing syntax supports specialized functionality such as:
+- Declaring max/min height or width
+```xml
+<Grid MaxHeight="600" MaxWidth="600">
+  ...
+</Grid>
+```
+
+- Data binding for row/height values:
+```xml
+<Grid>
+  <Grid.RowDefinitions>
+    <RowDefinition Height="{x:Bind sampleDataValue}" />
+    ...
+  </Grid.RowDefinitions>
+</Grid>
+```
+Grid's usability, intuitiveness, and developer satsifaction will be improved upon by utilizing a more succinct syntax, as will be described throughout this spec.
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+A new XAML language feature that allows read-only collection-type properties to be created by assigning the CreateFromString attribute. The CreateFromString attribute will allow these read-only collection-type properties to be defined/created from a comma-delimited string. In terms of Grid, this means that the ColumnDefinitions and RowDefinitions properties are able to be defined by providing a comma-delimited string of ColumnDefinitions (Width values) and a comma-delimited string of RowDefinitions (Height values), respectively. However, this feature is also applicable to other scenarios within the XAML language where collection-typed read-only properties are created (see Remarks for examples of such scenarios). 
```

Comment:
Clarification: this new feature's purpose is to "allow the initialization of collection-type properties (even if they are read-only) using element attribute syntax". The CreateFromString attribute is one potential way in which a collection type might take advantage of this feature.

Reply:
Fixed.